### ISSUE TITLE
Is it possible for mamba to disable lockfiles?

### ISSUE THREAD

**Issue by shaoqy1121** [mamba-org/mamba#3609]
In the next release it will be possible to disable lockfiles

_Originally posted by @jonashaag in https://github.com/mamba-org/mamba/issues/1515#issuecomment-1262758432_
            
When I set my pkgs_dir on a shared drive and it always reports"error    libmamba Could not set lock"

**Comment by shaoqy1121** [comment]
(base) qiyu@ubuntu:~$ mamba --version
mamba 1.5.10
conda 24.9.2
Is it possible to modify condarc or mambarc to allow mamba to install packages properly on shared drivers?

**Comment by Hind-M** [comment]
Yes you can use `mamba config set use_lockfiles False` as mentioned [here](https://mamba.readthedocs.io/en/latest/user_guide/troubleshooting.html#hangs-during-package-installation-on-nfs-network-file-systems).
Can you please confirm that this fixes your problem after trying it out?
Thanks!
